site/docs/how-to/telemetry.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 4
+slug: '/telemetry'
+---
+
+# Telemetry
+
+Telemetry is enabled by default when you run IaSQL locally for us to understand how you use the product and help us improve it. We use the [Posthog](https://posthog.com), which like IaSQL, is open source. If you wish to turn off telemetry you can simply set the `IASQL_TELEMETRY` flag to `off` when [starting](../getting-started.mdx) the docker container.
+
+```bash
+IASQL_TELEMETRY=off docker run -p 9876:9876 -p 5432:5432 --name iasql iasql/iasql
```

Comment:
I don't think it works this way. I think it needs to be:

```suggestion
docker run -e IASQL_TELEMETRY=off -p 9876:9876 -p 5432:5432 --name iasql iasql/iasql
```

Reply:
No host's environment variables won't be passed to the container:
<img width="691" alt="image" src="https://user-images.githubusercontent.com/5357435/220320481-8f60d14d-df1e-41f6-9aec-7822d3c67c31.png">

Right way as @dfellis suggested:
<img width="683" alt="image" src="https://user-images.githubusercontent.com/5357435/220320743-5737eff6-1f0e-48db-8a2c-a10ff7670213.png">